GETTING-STARTED.md

```
@@ -22,6 +22,29 @@ OpenSearch is used for full-text search and document storage. PostgreSQL is used
 ### Parsing
 The data for the dictionary is only available as semi-structured CSV or PDFs used for printing. Python scripts are used to parse those files to make them compatible and searchable with OpenSearch. [Read more](./parsing/README.md)
 
+## Code Style
+
+To keep pull requests focused on real changes (and not formatting noise), the TypeScript code in `./frontend/` and `./e2e/` is formatted with [Prettier](https://prettier.io/) and linted with [ESLint](https://eslint.org/).
+
+Install the root tooling once (from the repository root):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)